config/environments/production.rb

```diff
@@ -57,7 +57,11 @@
   # config.cache_store = :mem_cache_store
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  config.action_controller.asset_host = ENV['GOVUK_ASSET_HOST']
+  config.action_controller.asset_host = Proc.new do |source, request = nil, *_|
```

Comment:
It looks like this has a default value of `nil` for `request`, but it that happens the line below will fall over. Is that correct, and if so, does that mean the `nil` default can disappear?